modules/ob-release-notes-1-0.adoc

```diff
@@ -0,0 +1,46 @@
+// This module is included in the following assemblies:
+// * builds/ob-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="ob-release-notes-1-0_{context}"]
+= Release notes for {builds-title} General Availability 1.0
+
+{builds-title} General Availability (GA) 1.0 is now available on {ocp-product-title} 4.14.
+
+[id="new-features-1-0_{context}"]
+== New features
+
+The following sections highlight what is new in {builds-title} 1.0.
+
+[id="builds-new-features-1-0_{context}"]
+=== Builds
+
+* This release supports the `buildah` and `source-to-image` build strategies.
++
+[NOTE]
+====
+The `buildpacks` build strategy is currently in Developer Preview. It is planned to make this strategy generally available for use in a future release.
+====
+
+* With this release, you can define volume mounts in your `BuildStartegy` resource. When defining a strategy, you can declare volumes which can be shared across build steps and build runs. You can also declare the mount point for the volume which is fixed across all build steps. Volumes can either have a fixed volume source, or an overridable volume source that you can set in a `Build` or `BuildRun` resource.
```

Comment:
I'm unsure if "overridable" is an official word. 

or an volume source that you override by setting in a `Build` or `BuildRun` resource.

Reply:
I guess overridable volume source conveys the exact meaning and reflects technical accuracy.